reactor-core/src/main/java/reactor/core/publisher/DirectProcessor.java

```diff
@@ -80,7 +80,7 @@
  *
  * @param <T> the input and output value type
  * @deprecated To be removed in 3.5, prefer clear cut usage of {@link Sinks}:
- * <pre>Sinks.many().multicast().onBackpressureError();</pre> for example.
+ * <pre><code>Sinks.many().multicast().onBackpressureError();</code></pre>.
```

Comment:
nitpick: generally in javadocs oneliners like that are added through a `{@code xxx}` tag rather than html `<pre><code>`.

Reply:
actually, now that I've looked into the actual rendering the blocks make it add a newline, which is kinda good. I'll just edit the PR to remove the dot at the end (which also causes a newline, less desirable)